vgstation13.dme

```diff
@@ -2756,7 +2756,7 @@
 #include "maprendering\maprendering.dm"
 #include "maps\_map.dm"
 #include "maps\_map_override.dm"
-#include "maps\lowfatbagel.dm"
+#include "maps\tgstation.dm"
```

Comment:
Huh? Why was this changed, but also why was it lowfatbagel before?

Reply:
Unless I'm adding new files I usually don't even pay attention to the dme honestly